test/contribution.test.js

```diff
@@ -76,3 +77,38 @@ test('fetch contribution data via async/await with CORS', async done => {
   expect(typeof data.contributions).toBe('number');
   done();
 });
+
+test('fetch contribution data for unknown user via callback without CORS', done => {
```

Comment:
Can the test strings be renamed to "invalid user" instead of "unknown user" too?